lib/doc/worksheet.js

```diff
@@ -867,8 +867,25 @@ class Worksheet {
     this._media = value.media.map(medium => new Image(this, medium));
     this.sheetProtection = value.sheetProtection;
     this.tables = value.tables.reduce((tables, table) => {
-      const t = new Table();
+      const t = new Table(this, null);
+      table.ref = table.tableRef;
+      table.headerRow = true;
```

Comment:
This is actually a corruption as well. The bug is in that Excel gives `headerRowCount="0"` when there is a header anyway, and the presence of `headerRowCount` is the difference in the xml of the table. The uncorrupt fix would be to test for presence of the attribute in https://github.com/exceljs/exceljs/blob/master/lib/xlsx/xform/table/table-xform.js#L69

Otherwise tables that are hiding the header will end up skipping the first line